build/about.html

```diff
@@ -0,0 +1,87 @@
+            <li class="active"><a href="about.html">About</a></li>
+            <li><a href="support.html">Support</a></li>
+            <!--<li class="dropdown">-->
+            <!--<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>-->
+            <!--<ul class="dropdown-menu">-->
+            <!--<li><a href="#">Action</a></li>-->
+            <!--<li><a href="#">Another action</a></li>-->
+            <!--<li><a href="#">Something else here</a></li>-->
+            <!--<li role="separator" class="divider"></li>-->
+            <!--<li><a href="#">Separated link</a></li>-->
+            <!--<li role="separator" class="divider"></li>-->
+            <!--<li><a href="#">One more separated link</a></li>-->
+            <!--</ul>-->
+            <!--</li>-->
+          </ul>
+          <div class="col-sm-3">
+            <form class="navbar-form" role="search">
+              <div class="input-group nav-searched-curved">
+                <input type="text" class="form-control" placeholder="Search" name="srch-term" id="srch-term">
+                <div class="input-group-btn">
+                  <button class="btn btn-default" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+                </div>
+              </div>
+            </form>
+          </div>
+          <ul class="nav navbar-nav navbar-right">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Theo Proulx <span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a href="myprofile.html">Profile</a></li>
+                <!--<li><a href="#">Manage Settings</a></li>-->
+                <!--<li><a href="#">Notification Preferences</a></li>-->
+                <li role="separator" class="divider"></li>
+                <li><a href="#">Sign Out</a></li>
+              </ul>
+            </li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+    <div>
+      <center>
+        <h1>-Our Mission-</h1>
+        <p>To predict future game prices</p>
+        <h2>About</h2>
+        <div style="width:600px;height:200px;border:1px solid #000;">At StarVestments, our goal for our members is to accurately predict an up and coming game that is readily available on steam.  We want to ensure that all gamers can have access to their favorite games as fast as possible, and as cheap as possible.  Our algorithm to compute future prices involves the total amount of players online for that certain game, added to the number of online members with the game in his or her wishlist, and divide that sum by its current cost.  We believe that our web application can potentially change the landscape of the gaming community by bringing the best value in steam's market.</div>
```

Comment:
Im merging this for now @willjeong204 but if you have time:
* please move this inline style to a custom css file.
* round the edges of your border with the css property `border-radius: 20px` (site wide standard)